web-client/src/primitives/transaction_builder.rs

```diff
@@ -251,7 +256,43 @@ impl TransactionBuilder {
         Ok(Transaction::from_native(tx))
     }
 
-    // pub fn new_create_validator()
+    /// Registers a new validator in the staking contract.
+    ///
+    /// The returned transaction is not yet signed. You can sign it e.g. with `tx.sign(keyPair)`.
+    ///
+    /// Throws when the numbers given for fee do not fit within a u64 or the networkId is unknown.
+
```

Comment:
```suggestion
    /// Throws when the fee does not fit within a u64 or the `networkId` is unknown.
```
Some small suggestion for the phrasing and could you also remove the extra blank line as suggested by @sisou?